docs/modules/ROOT/pages/documentation/payara-server/request-tracing-service/tracing-remote-ejbs.adoc

```diff
@@ -0,0 +1,151 @@
+[[tracing-remote-ejbs]]
```

Comment:
This new file needs to be added to nav.adoc, otherwise it's not visible in the generated documentation.